tidy3d/components/data/dataset.py

```diff
@@ -388,6 +396,20 @@ def n_group(self):
             )
         return self.n_group_raw
 
+    @property
+    def dispersion(self) -> ModeDispersionDataArray:
+        """Dispersion parameter.
+
+        D = -lda / C_0 d2n/d(lda)2
```

Comment:
I think @daquinteroflex should make a note to update this with a proper equation in the style of his api docs update? Or just tell us how to do it already directly here? We already have equations like 

```
    .. math::

        \\epsilon(f) = \\epsilon_\\infty + \\sum_i
        \\frac{\\Delta\\epsilon_i f_i^2}{f_i^2 - 2jf\\delta_i - f^2}
```

is this how this should be reformatted?

Reply:
Yep adding it this way is correct! Here you have some examples: this line https://github.com/flexcompute/tidy3d/blob/2f147a3e28195097203746ff8621913906a586bb/tidy3d/components/medium.py#L2042 rendered [here](https://flexcompute-tidy3d-docs-demo.readthedocs-hosted.com/en/repo_merge_no_history/api/_autosummary/tidy3d.PoleResidue.html#tidy3d.PoleResidue)

Reply:
I'll update it on my branch in any case!

Reply:
Sounds good. I was not sure if we wanted to favor the web rendering or the terminal help. Changed!

Reply:
Hmm what is the `r` for in front of the `"""` of the docstring?

Reply:
That's for a raw string, so that we don't have to escape the backslashes.